src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSymbolDeclarationService.vb

```diff
@@ -55,7 +55,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         begin = DirectCast(parent, EventBlockSyntax).EventStatement
 
                     Case SyntaxKind.VariableDeclarator
-                        begin = node
+                        If DirectCast(parent, VariableDeclaratorSyntax).Names.Count = 1 Then
+                            begin = node
+                        End If
```

Comment:
what's going on here.

Reply:
Symbol declaration service was incorrectly returning the entire field declaration syntax as the declaring syntax for a declaration with more than one fields being declared. It should do so only if the field declaration declares a single variable.

---
In reply to: [213423589](https://github.com/dotnet/roslyn/pull/29511#discussion_r213423589) [](ancestors = 213423589)